### PR TITLE
Platform integration

### PR DESCRIPTION
**Pull request by pbochis**

### CODE REVIEW

File: app/elements/coduno-iron-ajax.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+				}
+			},
+			generateRequest: function(){
+				console.log(this.url);
+				if (localStorage.authorization){
+					this.headers.Authorization = localStorage.authorization;
+				}
+				this.$.ajaxRequest.generateRequest();
+			},
+			_onResponse: function(req){
```
Comment (victorbalan) [inline app/elements/coduno-iron-ajax.html:63]: this is not needed you can specify {onResponse} directly afaik

File: app/elements/app-element.html [hunk 2]
```diff
@@ -11,27 +12,22 @@
...
 			debounce-duration="1000"
-			content-type="application/json"
 			with-credentials="true"
 			on-error="onResultError"
 			>
-		</iron-ajax>
+		</coduno-iron-ajax>
 
-		<iron-ajax
+		<codunoiron-ajax
```
Comment (lorenzleutgeb) [inline app/elements/app-element.html]: Missing '-'

File: app/elements/coduno-iron-ajax.html [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+					notify: true
+				},
+				debounceDuration: {
+					type: Number,
+					notify: true,
+					value: 0
+				}
+			},
+			generateRequest: function(){
+				console.log(this.url);
```
Comment (lorenzleutgeb) [inline app/elements/coduno-iron-ajax.html]: d

File: app/elements/coduno-iron-ajax.html [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+				if (localStorage.authorization){
+					this.headers.Authorization = localStorage.authorization;
+				}
+				this.$.ajaxRequest.generateRequest();
+			},
+			_onResponse: function(req){
+				console.log(req.detail.response);
```
Comment (lorenzleutgeb) [inline app/elements/coduno-iron-ajax.html]: d

File: app/elements/coduno-iron-ajax.html (hunk 1 above)
Comment (pbochis) [inline app/elements/coduno-iron-ajax.html:63]: No, it doesn't work. Just tested

**Pull request merged**